server/sonar-server/src/main/java/org/sonar/server/component/ws/ProjectMeasuresQueryFactory.java

```diff
@@ -39,6 +39,7 @@
 class ProjectMeasuresQueryFactory {
 
   public static final String IS_FAVORITE_CRITERION = "isFavorite";
+  public static final String QUERY_KEY = "q";
```

Comment:
I prefer query that q:

- more explicit
- might confuse with another q parameter that means something else